lib/lifecycles/bump.js

```diff
@@ -20,16 +20,45 @@ async function Bump (args, version) {
   configsToUpdate = {}
 
   if (args.skip.bump) return version
+
+  if (args.releaseAs && !(['major', 'minor', 'patch'].includes(args.releaseAs.toLowerCase()) || semver.valid(args.releaseAs))) {
+    throw Error("releaseAs must be one of 'major', 'minor' or 'patch', or a valid semvar version.")
```

Comment:
```suggestion
    throw new Error("releaseAs must be one of 'major', 'minor' or 'patch', or a valid semvar version.")
```

(unless I'm missing something)

Reply:
Nope, I'm just too used to typescript yelling at me if I miss a `new`